qiskit/providers/aer/jobs/aerjobset.py

```diff
@@ -335,9 +335,9 @@ def _combine_results(self,
 
         if self._start_time:
             _time_taken = self._end_time - self._start_time
-            combined_result.time_taken = _time_taken.total_seconds()
```

Comment:
I guess that `time_taken` should have  `_time_taken.total_seconds()`, too, as `AerJob` has.

Reply:
I added the line to set the time to `combined_result.time_taken `.